File: C#/ConsoleApplication1/Car.cs
using System.Collections.Generic;


namespace ConsoleApplication1
{
    public class Car
    {
        protected int Wheels;

        protected Vector3<float> Position;
        protected Vector4<float> Color;

        protected Driver Driver;
        protected List<Passenger> Passengers;

        public virtual void Draw()
        {
            // Draw To Screen
        }               

        public virtual void Move(Vector3<float> NewPosition)
        {
            Position = NewPosition;
        }

        public Car()
        {
            Position = new Vector3<float>(0.0f, 0.0f, 0.0f);
        }
        ~Car()
        {

        }
    }
}

File: C++/Inheritance/Car.cpp
#include "Car.h"



void Car::Draw()
{
	// Draw To Screen
}

void Car::Move(Vector3<float> NewPosition)
{
	Position = NewPosition;
}


Car::Car()
{
	Position = Vector3<float>(0.0f, 0.0f, 0.0f);
}

Car::~Car()
{

}

